src/gluonnlp/embedding/token_embedding.py

```diff
@@ -173,23 +173,15 @@ class TokenEmbedding(object):
         tokens can be updated.
     unknown_lookup : object subscriptable with list of tokens returning nd.NDarray, default None
         If not None, unknown_lookup[tokens] is called for any unknown tokens.
```

Comment:
Descriptions of `unknown_lookup` may be a bit vague for people without background information. Can you improve it with better illustrations?

Reply:
I added some more comments